Gemfile

```diff
@@ -14,6 +14,7 @@ gem 'nokogiri', '1.6.6.4'
 gem 'plek', '~> 2.0'
 gem 'rack_strip_client_ip', '0.0.1'
 gem 'rails', '4.2.7.1'
+gem 'govuk_sidekiq'
```

Comment:
This will violate the `Bundler/OrderedGems` cop. The gems should be listed in alphabetical order.